### ISSUE TITLE
Cumulative time

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#41]
I was looking at the code for the schedulers and I did not see a point where the `cumulative_time` gets changed. I also looked at the rest of the repository and the only point where `cumulative_time` is being calculated is at the `image_trainer`, but still, not being passed to the scheduler.

https://github.com/automl/Auto-PyTorch/blob/d60369227e96fef922dcc79005b839fefb972e76/autoPyTorch/components/lr_scheduler/lr_schedulers.py#L392
